src/charts/mapd-table.js

```diff
@@ -344,8 +344,14 @@ export default function mapdTable(parent, chartGroup) {
             customFormatter = _chart.valueFormatter()
           }
 
-          const key = (val && val[0] && val[0].isExtract) ? null : col.measureName || col.expression
-          return customFormatter && customFormatter(val, key) || formatDataValue(val)
+          const key =
+            val && val[0] && val[0].isExtract
+              ? null
+              : col.measureName || col.expression
+          return (
+            (customFormatter && customFormatter(val, key)) ||
+            formatDataValue(val)
+          )
```

Comment:
This is a stylistic suggestion, so don't feel obligated.

What about defining customFormatter as:

`customFormatter ||= _chart.valueFormatter() || (val, key) => formatDataValue(val)`

Then the later logic can be simplified a lot. e.g.:
`(customFormatter && customFormatter(val, key)) || formatDataValue(val)`
becomes:
customFormatter(val, key)